frontend/src/component/Workspace/ContentItemHeader.jsx

```diff
@@ -15,11 +16,23 @@ export const FileItemHeader = props => {
           <div className='content__header__search__path'>
             {props.t('Path')}
           </div>
-          <div className='content__header__search__modif'>
+          <div className='content__header__search__modification'>
             {props.t('Last Modification')}
           </div>
         </div>
       )}
+      {props.showLastModification && !props.showSearchDetails && (
```

Comment:
It would be better to define usage for the component and having only one props.
eg.: search and contentList

It might require to duplicate a bit of html but it separates responsibilities and avoid complex boolean expressions.

Another approach could be to do composition. I don't know if it's better, haven't looked enough at the code
eg.:
```jsx
<FileItemHeader><SearchHeader /></FileItemHeader>
```
and
```jsx
<FileItemHeader><ContentItemHeader /></FileItemHeader>
```